core-primitives/api-client-extensions/src/chain.rs

```diff
@@ -33,6 +36,17 @@ where
 		self.get_signed_block(hash)
 	}
 
+	fn get_blocks(&self, from: u32, to: u32) -> ApiResult<Vec<SignedBlock>> {
+		let mut blocks = Vec::<SignedBlock>::new();
+
+		for n in from..=to {
+			if let Some(block) = self.get_signed_block_by_num(Some(n))? {
+				blocks.push(block);
+			}
+		}
+		Ok(blocks)
+	}
```

Comment:
I think it is better to return an `ApiClientError::Other("'from' must be <= 'to'".into())` than to silently return an empty vec. Or we should at least document that behaviour.

Reply:
Agreed 👍